validator/entrypoint.sh

```diff
@@ -49,7 +49,7 @@ exec -c validator --prater \
   --grpc-gateway-host=0.0.0.0 \
   --grpc-gateway-port="$VALIDATOR_PORT" \
   --grpc-gateway-corsdomain=http://0.0.0.0:"$VALIDATOR_PORT" \
-  --graffiti="$GRAFFITI" \
+  --graffiti="${GRAFFITI:0:32}" \
```

Comment:
How those that handle emojis, non-western characters, accented characters, etc?

Reply:
Yes, you're right. I will fix this.